src/table/table-body.jsx

```diff
@@ -384,16 +384,16 @@ const TableBody = React.createClass({
 
   _onCellClick(e, rowNumber, columnNumber) {
     e.stopPropagation();
-    if (this.props.onCellClick) this.props.onCellClick(rowNumber, this._getColumnId(columnNumber));
+    if (this.props.onCellClick) this.props.onCellClick(rowNumber, this._getColumnId(columnNumber), e);
```

Comment:
I think that it would be better to use `event` instead of `e`. And that the event should be at the first argument.